### DISCUSSION TITLE
Have any plans to install the discord.types package separately?

### DISCUSSION THREAD

**Discussion by SaidBySolo** [Rapptz/discord.py#8137] [Ideas] [answered]
I'm implementing a bot without using a gateway.

I need a type hint for the response.

``discord.types`` do what I want them to do great.

I can install discord.py to use it, but I don't need the full library of discord.py.

Do you have any plans for this?

**Comment by Rapptz** [comment, marked as answer]
Unfortunately, I do not. Maintaining these separately would be kind of cumbersome so I kept it as a monorepo structure.

**Reply by SaidBySolo** [reply]
Thank you for quick response.

Let's find another way.

**Reply by RGBCube** [reply]
can't you achieve this using git submodules?

**Reply by Rapptz** [reply]
Yes but that's still two separate repositories that contributors and I would have to manage.
